modules/renter/download_test.go

```diff
@@ -4,42 +4,39 @@ import (
 	"bytes"
 	"crypto/rand"
 	"io"
-	"sync"
 	"testing"
 	"time"
 
 	"github.com/NebulousLabs/Sia/crypto"
-	"github.com/NebulousLabs/Sia/modules"
 )
 
-type testHost struct {
-	ip        modules.NetAddress
+// a testFetcher simulates a host. It implements the fetcher interface.
```

Comment:
// a testFetcher
=>
// testFetcher

go lint would complain about this if it was an exported function